packages/client/bin/cli.ts

```diff
@@ -207,7 +207,8 @@ const args: ClientOpts = yargs(hideBin(process.argv))
   .option('rpcDebug', {
     describe:
       'Additionally log complete RPC calls filtered by name (prefix), e.g.: "eth,engine_getPayload" (use "all" for all methods).',
-    default: '',
+    coerce: (arg: string) => (arg === '' ? 'all' : arg),
```

Comment:
When I initially changed `rpcDebug` to allow for indicating a specific package for full request/response log output, I included a description with details about the need for `logLevel` being set to `debug` and also included choices which allowed for boolean options or a specific module to be enabled by name, as well as a default of `false`: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3102/commits/b196a131e92c80a30814031d4885b9233e7dfdef. But now that we can pass in a comma separated string that could also include specific function names (e.g. `eth,engine_getPayload`) I think that we can look into using a yargs `check` function for `rpcDebug` to first validate the input and throw helpful error messages so users aren't confused as well as extend our description to include more usage details and maybe even some examples.